.travis.yml

```diff
@@ -1,11 +1,11 @@
 language: scala
 scala:
-  - 2.11.8
+  - 2.12.10
 sudo: false
 jdk:
-  oraclejdk8
+  openjdk8
```

Comment:
Thanks @wangyum. Can we try to add `dist: trusty` first before chaning to openjdk? I read https://travis-ci.community/t/install-of-oracle-jdk-8-failing/3038. If that doesn't work let's do it for openjdk.

Reply:
It works.